AGENT_CHANGELOG.md

```diff
@@ -1,8 +1,46 @@
-## Datadog Agent version [7.49.1](https://github.com/DataDog/datadog-agent/blob/master/CHANGELOG.rst#7491)
```

Comment:
Question to reviewers: should we be worried that release 7.49.1 disappeared from this file?

Reply:
Yes we should bring it back and either have a look at the issue or create a card imo